docs/aws/nRFConnectForCloudAPI.rst

```diff
@@ -0,0 +1,55 @@
+.. _aws-nrf-connect-for-cloud-api:
+
+nRF Connect for Cloud API
+#########################
+
+You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for the application using the `nRF Connect for Cloud's geolocation API <https://api.nrfcloud.com/v1/#operation/GetSingleCellLocation>`_  in your deployment.
+
+Creating an API device
+**********************
+
+The nRF Connect for Cloud API requires a device identifier for some calls.
+It is recommended to create a new device for the API operations.
+
+Create a new device using `httpie <https://httpie.io/>`_:
+
+.. code-block:: bash
+
+    API_KEY="<API Key>" # provide your nRF Connect for Cloud API key
+    DEVICE_ID=`uuidgen`
+    ENDPOINT=`http https://api.nrfcloud.com/v1/account Authorization:"Bearer ${API_KEY}" | jq -r '.mqttEndpoint'`
+    http POST https://api.nrfcloud.com/v1/devices/${DEVICE_ID}/certificates \
+        Authorization:"Bearer ${API_KEY}" > -v <<< 1234 \
+        | jq --arg endpoint ${ENDPOINT} '. + {brokerHostname: $endpoint}' > ${DEVICE_ID}.json
+
+Connect the device using the device simulator:
+
+.. code-block:: bash
+
+    npm exec -- @nordicsemiconductor/asset-tracker-cloud-device-simulator-aws \
+        ${DEVICE_ID}.json
+
+Once it prints ``connected``, you can terminate the simulator.
+You can ignore the ``disconnected!`` warnings.
+
+Now associate the device to your account:
+
+.. code-block:: bash
+
+    http PUT https://api.nrfcloud.com/v1/association/${DEVICE_ID} \
+        Authorization:"Bearer ${API_KEY}" -v <<< 1234
+
+You are now ready to use the device for calls to the nRF Connect for Cloud geolocation API.
+
+Enabling the nRF Connect for Cloud geolocation API
+**************************************************
+
+To use nRF Connect for Cloud's geolocation API, configure your API key and API device using the CLI and redeploy the stack using the following commands:
+
+.. code-block:: bash
+
+    node cli configure-api cellGeoLocation nrfconnectforcloud apiKey "<API Key>"
+    node cli configure-api cellGeoLocation nrfconnectforcloud apiDevice "<API device>"
+    npx cdk -c nrfconnectforcloud=1 deploy '*'
```

Comment:
As in my list this week: "Don't use arrow brackets. Instead, use the .. parsed-literal:: directive for code fragments that include variable names,  for example installation paths or board names. With this directive, you can apply the emphasis markup for variables, for example *API key* or *API device*.

Reply:
Can you provide the correct markup please?

Reply:
Instead of .. code-block:: bash, use ..parsed-literal and then have "*API key*". Use the normal emphasis markup, meaning an asterisk on both sides of the variable.

Reply:
I don't like that parsed-literal removes all other syntax highlighting. It worsens the usability of this code section in my opinion. The bash syntax highlighting makes the variables stand out as well.

`.. code-block:: bash`

![20210414T143140](https://user-images.githubusercontent.com/188915/114710679-4f559f80-9d2e-11eb-824d-82d94ada14de.png)

`.. parsed-literal::`

![20210414T143140](https://user-images.githubusercontent.com/188915/114710732-5bd9f800-9d2e-11eb-9297-1d994a2bfbb9.png)

Please advise.

Reply:
Well, I have to agree that the syntax highlighting works better with code-block + language type. However, let's stick to the rules I referred to as far as possible and where applicable.

Reply:
As you wish.